stackscli.yml

```diff
@@ -0,0 +1,206 @@
+# Configuration file for setting up the project
+
+framework:
+  name: java
+  commands:
+    - name: java
+      version: ">= 11"
+
+# Pipeline files
+pipeline:
+  - type: azdo
+    files:
+      - name: build
+        path: build/azDevOps/azure/azure-pipelines-javaspring-k8s.yml
+      - name: variable
+        path: build/azDevOps/azure/azuredevops-vars.yml
+    replacements:
+      - pattern: ^.*stacks-credentials-nonprod-kv$
+        value: ""
+      - pattern: ^.*amido-stacks-infra-credentials-nonprod$
+        value: ""
+      - pattern: ^.*amido-stacks-infra-credentials-prod$
+        value: ""
+      - pattern: ^.*stacks-credentials-prod-kv$
+        value: ""
+      - pattern: ^.*stacks-credentials-nonprod-kv$
+        value: ""
+      - pattern: ^.*amido-stacks-java-api$
+        value: ""
+      - pattern: ^.*amido-stacks-java-api-cqrs$
+        value: ""
+
+# The init stage are things that are required to run before the template is run
+init:
+  operations:
+    # Copy pipelines for build and deploy
+    - action: cmd
+      cmd: mvn
+      args: process-resources
+              -f stacksclipom.xml
+              -P copy-devops-resources
+              -Dtarget.dir={{ .Project.Directory.WorkingDir }}
+              --quiet
+      desc: Coping build and deploy pipelines into the project structure
+
+    # Build archetype for the main code
+    - action: cmd
+      cmd: mvn
+      args: clean archetype:create-from-project
+              -f ./java/pom.xml
+              -DpropertyFile=./java/archetype.properties
+              --quiet
+      desc: Creating the archetype resources using the web-api source code
```

Comment:
project reference, this is cqrs not web-api :)

Reply:
Nice spot